src/frontends/common/include/openvino/frontend/input_model.hpp

```diff
@@ -36,10 +36,20 @@ namespace frontend {
 /// All editing requests affect the model representation that is held behind the scene
 /// successive method calls observe a new graph structure.
 class FRONTEND_API InputModel {
```

Comment:
am I right that as a side effect of this we cannot use dynamic_cast to FE-specific InputModel?

Reply:
right